common/interface.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __INTERFACE__
+#define __INTERFACE__
+
+#include <string>
+#include <net/if.h>
+
+namespace swss {
+
+inline bool isInterfaceNameLenOk(const std::string &ifaceName)
+{
+    if (ifaceName.length() >= IFNAMSIZ)
+    {
+        SWSS_LOG_ERROR("Invalid interface name %s length, it must not exceed %d characters", ifaceName.c_str(), IFNAMSIZ);
```

Comment:
@arfeigin this is a library helper function. It should not generate any logs but only provide an interface. It is customer (upper layer) code responsibility to treat the result as error or not and print the relevant message

Reply:
I get your point but I did this to avoid code repetitions as it would be handled in single place, such also can be found in db_connector.cpp, events_common.cpp/h, redisapi.h, and other swss-common files.